test/checks/missing_enable_comment_test.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require "test_helper"
+require "minitest/focus"
+
+class MissingEnableCommentTest < Minitest::Test
```

Comment:
Would it make sense to also look for `theme-check-enable` with missing `theme-check-disable`?

Reply:
I like the idea. I tried implementing this, but I'm struggling to come up with a good mechanism for checking this.

My naive approach was to keep a simple counter (+/-) based on # of disable/enable statements. If the count is > 0 at the end of the document, it means there's a superfluous `enable` comment.

However, that doesn't work when you have multiple sets of checks, or you granularly enable certain checks.

```liquid
{% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
Stuff
{% comment %}theme-check-enable TracerCheck{% endcomment %}
More stuff
{% comment %}theme-check-enable AnotherCheck{% endcomment %}
```

Finally, it seems like we ought to allow users to blanket-enable all checks at any given point, meaning we can't even rely upon the # of checks re-enabled granularly:

```liquid
{% comment %}theme-check-disable TracerCheck, AnotherCheck, MoreChecks{% endcomment %}
Stuff
{% comment %}theme-check-enable TracerCheck{% endcomment %}
Blanket re-enable:
{% comment %}theme-check-enable{% endcomment %}
```

I'm OK passing on this for now and considering an implementation down the line if we find one. Thoughts?

Reply:
Yes, you're right. Should be in another PR. Can you open an issue afterwards so we don't forget about this?